spec/player_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'player'
+require 'team'
+
+describe Player do
+
+  describe 'applying for a team' do
+
+    let(:player) { Player.new :female }
+    let(:team) { Team.new }
+
+    it 'is valid if the player is not already part of a team' do
+      expect(player).to be_hireable
+    end
+
+    it 'is not valid if the player is already part of a team' do
+      team.add_player player
```

Comment:
Why do you need a team here? What we're really testing is the #employ method on Player.